CHANGELOG.md

```diff
@@ -1,9 +1,70 @@
+
+#### Documentation Updates
+ * Document --enable-incomplete-feature possible values in "mypy --help" (Froger David, PR [16661](https://github.com/python/mypy/pull/16661))
+ * Update new type system discussion links (thomaswhaley, PR [16841](https://github.com/python/mypy/pull/16841))
+ * Docs: Add missing class instantiation to cheat sheet (Aleksi Tarvainen, PR [16817](https://github.com/python/mypy/pull/16817))
+ * Fix typo in getting_started.rst (zipperer, PR [16700](https://github.com/python/mypy/pull/16700))
+ * Document how evil `--no-strict-optional` is (Shantanu, PR [16731](https://github.com/python/mypy/pull/16731))
+ * Improve mypy daemon documentation note about local partial types (Makonnen Makonnen, PR [16782](https://github.com/python/mypy/pull/16782))
+ * Fix numbering error in docs (Stefanie Molin, PR [16838](https://github.com/python/mypy/pull/16838))
+ * Various docs improvements (Shantanu, PR [16836](https://github.com/python/mypy/pull/16836))
+
+#### Stubtest Improvements
+  * Stubtest will ignore private function/method parameters when they are missing from the stub.
+Private parameters names start with a single underscore and have a default
+(PR [16507](https://github.com/python/mypy/pull/16507)).
+ * Stubtest: ignore a new protocol dunder (#16895) (Marc Mueller, PR [16899](https://github.com/python/mypy/pull/16899))
+ * stubtest: Private parameters can be omitted (Sebastian Rittau, PR [16507](https://github.com/python/mypy/pull/16507))
+ * stubtest: Add support for setting enum members to "..." (Jelle Zijlstra, PR [16807](https://github.com/python/mypy/pull/16807))
+ * stubtest: adjust symtable logic (Shantanu, PR [16823](https://github.com/python/mypy/pull/16823))
+ * stubtest: fix pos-only handling in overload resolution (Shantanu, PR [16750](https://github.com/python/mypy/pull/16750))
+
+#### Stubgen Improvements
+ * stubgen: Fix crash on star unpack of TypeVarTuple (Ali Hamdan, PR [16869](https://github.com/python/mypy/pull/16869))
+ * Fix failing stubgen tests (Ali Hamdan, PR [16779](https://github.com/python/mypy/pull/16779))
+ * stubgen: use PEP 604 unions everywhere (Ali Hamdan, PR [16519](https://github.com/python/mypy/pull/16519))
+ * Improve stubgen tests (Fabian Keller, PR [16760](https://github.com/python/mypy/pull/16760))
+ * stubgen: Do not ignore property deleter (Ali Hamdan, PR [16781](https://github.com/python/mypy/pull/16781))
+ * Support type stub generation for `staticmethod` (WeilerMarcel, PR [14934](https://github.com/python/mypy/pull/14934))
+
+#### Plugin Improvements
+ * Add `alias` support to `field()` in `attrs` plugin (Nikita Sobolev, PR [16610](https://github.com/python/mypy/pull/16610))
+ * Improve attrs hashability detection (Tin Tvrtković, PR [16556](https://github.com/python/mypy/pull/16556))
+
+#### PEP 696 Support & PEP 646 Improvements
```

Comment:
```suggestion
#### Basic PEP 696 Support
```

Does it make sense to add a small paragraph explaining it? Something along the lines of:
```
This release will include basic support for TypeVar defaults [PEP-696](https://peps.python.org/pep-0696/).
The current implementation is still incomplete, notably these don't work yet
- Defaults for `TypeVarTuple`
- Defaults which reference other TypeVars
- The new syntax since mypy also doesn't support [PEP-695](https://peps.python.org/pep-0695/) yet.
```

An example from the PEP, which is probably the most common use and works already.
```py
class Bot: ...

BotT = TypeVar("BotT", bound=Bot, default=Bot)

class Context(Generic[BotT]):
    bot: BotT

class MyBot(Bot): ...

reveal_type(Context().bot)         # type is Bot  # notice this is not Any which is what it would be currently
reveal_type(Context[MyBot]().bot)  # type is MyBot
```

Reply:
Also maybe move this further up; I think new features deserve more prominence.